cmd/pipecd/server.go

```diff
@@ -465,8 +465,16 @@ func createFilestore(ctx context.Context, cfg *config.ControlPlaneSpec, logger *
 	}
 }
 
-func registerMetrics() {
-	r := prometheus.WrapRegistererWithPrefix("pipecd_server_", prometheus.DefaultRegisterer)
+func registerMetrics() *prometheus.Registry {
+	r := prometheus.NewRegistry()
+	wrapped := prometheus.WrapRegistererWith(map[string]string{
```

Comment:
I think this wrapped registerer should be used in L470,L410 instead of the r registerer 🤔

Reply:
good catch!